### ISSUE TITLE
Links in wiki return 403

### ISSUE THREAD

**Issue by OevreFlataeker** [merbanan/rtl_433#2951]
The links to googleusercontent on https://github.com/merbanan/rtl_433/wiki/OOK-Signal-Demodulation return error "403 Forbidden". Could the images maybe hosted on github directly?

**Comment by zuckschwerdt** [comment]
Good catch. Can somebody locate those two images? Maybe in the rtl_433 discussions list?
Otherwise I might recreate them later on, when pdv finally gets the update ;)

**Comment by OevreFlataeker** [comment]
Hm, it seems they are broken since 2018 at least... Also the Google Groups thread is broken: https://groups.google.com/g/rtl_433/c/B8uVi3bhr9U/m/xAR38ZZYBAAJ

**Comment by gdt** [comment]
It seems therefore the action is to drop the links.

**Comment by gdt** [comment]
I made a note i the wiki that the links are broken, with a request for the original bits.
